Documentation/deploy-master.md

```diff
@@ -301,6 +286,86 @@ spec:
       timeoutSeconds: 1
 ```
 
+### Run the per host calico agent
+The per host Calico agent runs on all hosts including the master node. It is responsible for enforcing networking policy and connecting the pods on that host to the flannel overlay network.
+
+When creating `/etc/systemd/system/calico-node.service`:
+
+* Replace `${ETCD_ENDPOINTS}`
+* Replace `${ADVERTISE_IP}` with this node's publicly routable IP.
+
+**/etc/systemd/system/calico-node.service**
+
+```yaml
+[Unit]
+Description=Calico per-host agent
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+Slice=machine.slice
+Environment=CALICO_DISABLE_FILE_LOGGING=true
+Environment=HOSTNAME=${ADVERTISE_IP}
+Environment=IP=${ADVERTISE_IP}
+Environment=FELIX_FELIXHOSTNAME=${ADVERTISE_IP}
+Environment=CALICO_NETWORKING=false
+Environment=NO_DEFAULT_POOLS=true
+Environment=ETCD_ENDPOINTS=${ETCD_ENDPOINTS}
+ExecStart=/usr/bin/rkt run --inherit-env --stage1-from-dir=stage1-fly.aci \
+--volume=modules,kind=host,source=/lib/modules,readOnly=false \
+--mount=volume=modules,target=/lib/modules \
+--trust-keys-from-https quay.io/calico/node:v0.19.0
+
+KillMode=mixed
+Restart=always
+TimeoutStartSec=0
+
+[Install]
+WantedBy=multi-user.target
+```
+
+### Set Up the policy-agent Pod
+
+The policy agent is the last major piece of the our master node. It monitors the API for changes related to network policy and configures Calico to implement that policy.
+
+When creating `/etc/kubernetes/manifests/policy-agent.yaml`:
+
+* Replace `${ETCD_ENDPOINTS}`
+
+**/etc/kubernetes/manifests/policy-agent.yaml**
+
+```yaml
+apiVersion: v1
+kind: Pod 
+metadata:
+  name: calico-policy-agent
+  namespace: calico-system 
+spec:
+  hostNetwork: true
```

Comment:
Nope, the etcd endpoint is literally a port on the controller instance. My point was that (due to lack of network policy!!) you can hit that endpoint from any pod.

I've been reading through the source for policy agent- it seems that the default behavior will use service token secret mounted in at the conventional location and the in-cluster kubernetes service. 

[defaults for ca.crt and kuberentes service endpoint](https://github.com/projectcalico/k8s-policy/blob/c62fe7e113b013141eca808980907c0af7b0773f/constants.py#L4)
[default for service token](https://github.com/projectcalico/k8s-policy/blob/a8e439792e5f993dc910561c0ac5f4604b22960c/policy_agent.py#L437)
[how the defaults get applied](https://github.com/projectcalico/k8s-policy/blob/a8e439792e5f993dc910561c0ac5f4604b22960c/policy_agent.py#L35)

tldr; remove `K8S_API` env var and host-networking flag, and the policy-agent pod should automatically use the in-cluster k8s service correctly.